pkgs/development/python-modules/terminaltables3/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltables3";
+  version = "4.0.0";
+  pyproject = true;
```

According to https://pypi.org/project/terminaltables3/ the package requires Python >= 3.8, so let's disable older python versions:
```suggestion
  pyproject = true;
  disabled = pythonOlder "3.8";
```

---

good point

---

Not needed. Oldest Python in nixpkgs is 3.9

---

Yes, but it is still possible to add this package in an overlay to an olther nixpkgs version. I would therefore add it here anyway.

---

I also used to set disabled in other packages. It does make a lot of sense for the mentioned reasons. Custom downstream environments do exist.